app/Models/EntityNote.php

```diff
@@ -167,4 +169,14 @@ public function collapsed(): bool
     {
         return Arr::get($this->settings, 'collapsed', false);
     }
+
+    /**
+     * @return mixed
+     */
+    public function users()
```

Comment:
Same here, the name users is confusing to someone reading the code for the first time, especially since there are no comments in the code. editingUsers() would make more sense.